Sources/WWCalendarTimeSelector.swift

```diff
@@ -2348,12 +2348,12 @@ internal class WWCalendarRow: UIView {
                                 if comparisonDates.contains(date - 1.day) {
                                     ctx?.fill(CGRect(x: CGFloat(i - 1) * boxWidth, y: y, width: boxWidth / 2, height: maxConnectorSize))
                                 }
-                                if comparisonDates.contains(date + 1.day) {
+                                if comparisonDates.contains(date + 1.day) ||
+                                    date == comparisonDates.sorted().last {
                                     ctx?.fill(CGRect(x: CGFloat(i - 1) * boxWidth + boxWidth / 2, y: y, width: boxWidth / 2, height: maxConnectorSize))
                                 }
                                 ctx?.setFillColor(backgroundHighlightColor)
-                                if date == comparisonDates.sorted().first ||
-                                    date == comparisonDates.sorted().last {
+                                if date == comparisonDates.sorted().first {
```

Comment:
Wouldn't it be better to calculate `comparisonDates.sorted()` once?

Reply:
fixed